packages/backend/src/api/controllers/ControllerResult.ts

```diff
@@ -17,3 +17,7 @@ export interface ControllerRedirectResult {
   type: 'redirect'
   url: string
 }
+
+export interface ControllerCreatedResult {
```

Comment:
You can add it to the discriminated union `ControllerResult` on the top of the file